### DISCUSSION TITLE
GPU transform

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#674] [Q&A]
In the output of `python monai/transforms/utils.py`, a transform can run on GPU if it's 1 under `Torch?` , correct?
![image](https://user-images.githubusercontent.com/90806807/165416337-93d3a3aa-7323-4ada-b6d6-297eb4f7c33a.png)

**Comment by Nic-Ma** [comment]
Hi @hw-ju ,

Yes, you can convert data from numpy array to torch Tensor and run on GPU, refer to this example:
https://github.com/Project-MONAI/tutorials/blob/master/acceleration/distributed_training/brats_training_ddp.py#L198-L199

Thanks.

**Reply by hw-ju** [reply]
Thanks @Nic-Ma! 
Just to confirm, only torch Tensor can be run on GPU, numpy array can't, correct?

**Reply by rijobro** [reply]
Correct. In theory we could possibly use cupy for some of these but it's not something we've explored in great detail yet. Which transform is in numpy that you're hoping to do on the GPU?

**Reply by hw-ju** [reply]
Thanks @rijobro! I don't have those numpy transforms on my mind now, but will report here when they come up. :D
